src/it/MJAR-162/pom.xml

```diff
@@ -32,12 +32,17 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>3.8.1</version>
+      <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.3.0</version>
+      </plugin>
       <plugin>
```

Comment:
Is it needed for parent pom upgrade?

Reply:
it's needed as the empty test jar is not filtering the .gitignore file with the defaults.